third_party/xla_client/metrics.cc

```diff
@@ -194,12 +258,9 @@ string MetricFnTime(double value) {
     int precision;
     char fill;
   } const time_parts[] = {
-      {"d", 86400.0 * 1e9, 2, 0, '0'},
-      {"h", 1440.0 * 1e9, 2, 0, '0'},
-      {"m", 60.0 * 1e9, 2, 0, '0'},
-      {"s", 1e9, 2, 0, '0'},
-      {"ms", 1e6, 3, 0, '0'},
-      {"us", 1e3, 3, 3, '0'},
+      {"d", 86400.0 * 1e9, 2, 0, '0'}, {"h", 1440.0 * 1e9, 2, 0, '0'},
```

Comment:
Hmm, one more instance.
Alex, can you confirm you get the same format?

Reply:
No, I don't get the format changes.

Reply:
Which version of clang-format you have in your machine?
Are you using the Google provided one?

(pytorch) dlibenzi@dlibenzi2:~/google-git/pytorch/xla$ clang-format --version
clang-format version google3-trunk (trunk r347980)

Reply:
Emacs is using the system clang-format ... does not carry its own.
It simply calls out to that.
Weird.
Which edit are you using?

Reply:
Can you try:

clang-format --style=Google metrics.cc > metrics_google.cc
diff -u metrics.cc metrics_google.cc

Reply:
Ok, I get the same results as you. Let's stick to this command line for all formatting then.

Reply:
You should be able to tell your editor to run that command line.
They keys are 1) make sure the editor is using the system version and not its own cached copy 2) use --style=Google